packages/blockchain-wallet-v4/src/redux/kvStore/shapeShift/sagas.js

```diff
@@ -51,7 +51,6 @@ export default ({ api, networks }) => {
       const kv = KVStoreEntry.fromMetadataXpriv(mxpriv, typeId, networks.btc)
       yield put(A.fetchMetadataShapeshiftLoading())
       const newkv = yield callTask(api.fetchKVStore(kv))
-      yield put(A.fetchMetadataShapeshiftSuccess(newkv))
```

Comment:
hmm, strange one

Reply:
clearly an old bug

Reply:
Yeah its unnoticeable except in import/export. I would love to know why calling the actions in such quick succession (if that is the cause) results in the weird payload.

Reply:
this line is 8 months old, wonder how did it work all of this time

Reply:
Because most imports/exports do not rely on shapeshift metadata (only if the flow has gone to exchange -> order history probably). However, tons of ui depends on lockbox metadata